input/pagecontent/mrp.md

```diff
@@ -21,19 +21,19 @@ Note that the exchange of the reconciled medication list, indication of conversa
 
 |Resource Type| Profile Name                             | Link to Profile                            |
 |---|------------------------------------------|--------------------------------------------|
-|Coverage| QI Core Coverage Profile                 | [QI Core Coverage]                         |
-|Encounter| QI Core Encounter Profile                | [QI Core Encounter]                        |
-|Location| QI Core Location Profile                 | [QI Core Location]                         |
+|Coverage| QI-Core Coverage Profile                 | [QI-Core Coverage]                         |
```

Comment:
Same prior comments. These are specific profile names. use QICore instead to match the specific profile names.